Vagrant/Vagrantfile

```diff
@@ -4,71 +4,54 @@ Vagrant.configure("2") do |config|
     cfg.vm.box = "bento/ubuntu-20.04"
     cfg.vm.hostname = "logger"
     cfg.vm.provision :shell, path: "bootstrap.sh"
-    cfg.vm.network :private_network, ip: "192.168.38.105", gateway: "192.168.38.1", dns: "8.8.8.8"
-
-    cfg.vm.provider "vmware_desktop" do |v, override|
```

Comment:
Let's not remove VMware option. If somebody has VMware instead of VirtualBox, then they should be able to run this on VMware.

Reply:
Alright, I will add VMware back

Reply:
The file has been updated